docs/source/config.json

```diff
@@ -22,7 +22,11 @@
       "Logging": "/configuration/logging",
       "Header propagation": "/configuration/header-propagation",
       "Subgraph routing URLs": "/configuration/subgraph-routing-urls",
-      "Telemetry": "/configuration/telemetry"
+      "Telemetry": {
+        "Overview": "/configuration/telemetry",
+        "OpenTelemetry": "/configuration/open-telemetry",
+        "Spaceport": "/configuration/spaceport"
+      },
```

Comment:
@BrynCooke I would be in favor of avoiding a dedicated page for Spaceport right now.  I think it is quite an  advanced use-case at the moment and I think could fit into the bottom of a page that was just "Telemetry" titled, after a section called OpenTelemetry that lives on the same page.  Thoughts?

Reply:
Yeah, it's still a dedicated page